util/timeline_filter/timelineFilter.py

```diff
@@ -204,6 +180,12 @@ def loadOppositeEventsTable():
     return dicOppEvents
 
 def parseXml(filepath, timeline):
+    try:
+        fh = codecs.open(filepath, 'r', encoding='utf-8')
```

Comment:
it's this additional parsing need? I was thinking that elementree was able to check itself, but maybe I am wrong.

Reply:
recent versions are able to detect the formating, but not old ones, and we have to be able to run on python2

Reply:
ok, thx!